libraries/rush-lib/src/logic/buildCache/ProjectBuildCache.ts

```diff
@@ -79,10 +78,10 @@ export class ProjectBuildCache {
     return this._cacheId;
   }
 
-  public static async tryGetProjectBuildCache(
+  public static async getProjectBuildCache(
```

Comment:
```suggestion
  public static async getProjectBuildCacheAsync(
```

Reply:
Actually, this doesn't need to be async anymore.

Reply:
That was why the rename, but I forgot to de-async it.